front/tests/003-study-management.spec.ts

```
@@ -72,14 +69,14 @@ test.describe('Validate the Study creation workflow', () => {
 
   /** *************** Test 2 **************** */
   test('Update an existing study', async ({ page }) => {
-    const studyPage = new StudyPage(page);
+    const PROJECT_LANGUAGE = process.env.PROJECT_LANGUAGE || '';
```

what if `PROJECT_LANGUAGE = ''` ?

---

My bad, I missed that. Now that I’ve created a utility function, it should throw an error